code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -679,6 +679,7 @@
 		ADD_TRAIT(new_spawn, TRAIT_EXEMPT_HEALTH_EVENTS, GHOSTROLE_TRAIT)
 		ADD_TRAIT(new_spawn, TRAIT_NO_MIDROUND_ANTAG, GHOSTROLE_TRAIT) //The mob can't be made into a random antag, they are still eligible for ghost roles popups.
 		to_chat(new_spawn,"<span class='boldwarning'>Ghosting is free!</span>")
+		to_chat(new_spawn,"<span class='narsiesmall'>Be warned: People who opt out of EORG will come here. Do not make the area uninhabitable and do NOT commit EORG. This is a safe-zone. If you attack people in EORG, you will be banned for griefing.</span>")
```

Comment:
does it really have to be this bold

Reply:
Yes, admins are very annoyed with EORG ahelps and this removes any ambiguity of if a person is griefing or if they genuinely didn't know they weren't allowed to EORG in the safe zone.
After a month or so, I planned to make it smaller. But until everyone is used to it and it's set in stone, it's staying big so there's no confusion.
It's an extremely small price to pay for wasting five to ten minutes for 70+ people because of delayed round ends.